lib/bundler/audit/version.rb

```diff
@@ -18,6 +18,6 @@
 module Bundler
   module Audit
     # bundler-audit version
-    VERSION = '0.9.3'
+    VERSION = "0.10.0"
```

Comment:
Please don't bump the version in a PR. I will make a separate branch for 0.10.0 and do that.